client/messaging_contentstore/contentstore.py

```diff
@@ -0,0 +1,181 @@
+
+"""
+Client for Messaging Content Store HTTP services APIs.
+
+"""
+import requests
+import json
+
+
+class ContentStoreApiClient(object):
+
+    """
+    Client for Content Store API.
+
+    :param str auth_token:
+
+        An access token.
+
+    :param str api_url:
+        The full URL of the API. Defaults to
+        ``http://testserver/contentstore``.
+
+    """
+
+    def __init__(self, auth_token, api_url=None, session=None):
+        self.auth_token = auth_token
+        if api_url is None:
+            api_url = "http://testserver/contentstore"
+        self.api_url = api_url
+        self.headers = {
+            'Authorization': 'Token ' + auth_token,
+            'Content-Type': 'application/json'
+        }
+        if session is None:
+            session = requests.Session()
+        session.headers.update(self.headers)
+        self.session = session
+
+    def get_messagesets(self, params=None):
+        url = '%s/messageset/' % (self.api_url.rstrip('/'),)
+        result = self.session.get(url, params=params)
+        result.raise_for_status()
+        return result.json()
```

Comment:
Instead of repeating this, what about a function that takes the tail of the url (`/messageset/`) as an argument, and returns json, raising the correct http errors etc.